docs/core/generators/photonjs.md

```diff
@@ -70,11 +70,22 @@ This example shows the configuration of a Photon.js generator for local developm
 ```prisma
 generator photon {
     provider = "photonjs"
-    platforms = ["native", "linux-glibc-libssl1.0.2"] // For Lambda (Node 10) 
+    platforms = ["native", "linux-glibc-libssl1.0.2"] // For Lambda (Node 10)
     pinnedPlatform = env("PLATFORM")                  // Local: "native"; In production: "linux-glibc-libssl1.0.2"
 }
 ```
 
+## Compiling Custom Binary
+
+If a binary is not available for the platform you want to target. It is possible to compile the Prisma binaries on that platform. To compile a binary manually, please follow the following steps:
+
+1. Download the rust toolchain: https://www.rust-lang.org/tools/install
+2. Clone https://github.com/prisma/prisma
+3. Switch to `alpha` branch (during the preview period, we are using the `alpha` branch)
+4. Change the directory to `prisma/server/prisma-rs`
+5. Run `cargo build --release`
+6. The binaries should be available the `prisma/server/prisma-rs/target/release` folder, the name of the query engine binary is `prisma` and the migration engine binary is `migration-engine`.
```

Comment:
```suggestion
6. The binaries should be available in the `prisma/server/prisma-rs/target/release` folder, the name of the query engine binary is `prisma` and the migration engine binary is `migration-engine`.
```

What does the second have of this sentence, beginning with "the name of..." mean?